structuredefinitions/Extension-UKCore-SubscriptionNotificationHydration.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-UKCore-SubscriptionNotificationHydration" />
```

Comment:
Should be Extension-UKCore-SubscriptionContent

Reply:
As it is a backport I'd like to keep the name, id, title close to what the R5 element is named